### PR TITLE
Fix vulnerabilities

### PR DESCRIPTION
**Pull request by axbarsan**
* To get rid of the vulnerabilities in the old test setup, I replaced them with our one and only saviour: `jest`
* Ran the `jest-codemod` to automatically migrate all tests to `jest`
* Removed the testing step from CI, since we don't really have any tests, and empty tests make `jest` fail

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  "browser": {
-    "fs": false
+    "test": "./node_modules/.bin/jest"
   },
   "dependencies": {
     "superagent": "3.8.3"
   },
   "devDependencies": {
-    "mocha": "~2.3.4",
-    "sinon": "1.17.3",
```
Comment (stone-z) [inline package.json]: Nice, two fewer dependabot PRs to deal with :+1:

File: package-lock.json [hunk 2]
```diff
@@ -1,398 +1,4671 @@
 {
-  "name": "giantswarm-v4",
+  "name": "giantswarm-",
```
Comment (stone-z) [inline package-lock.json]: Is the trailing - intentional here?

**Comment by stone-z** [comment]
Why are the tests commented out?

File: package-lock.json (hunk 2 above)
Comment (axbarsan) [reply package-lock.json]: That's because for some reason it was like that in `package.json`. I updated it, though

**Comment by axbarsan** [comment]
> Why are the tests commented out?

Tests are commented out because they're mostly placeholders. Guess we're supposed to write our own?
```
// uncomment below and update the code to test the property clientKeyData
//var instane = new GiantSwarm.V4AddKeyPairResponse();
//expect(instance).to.be();
```

**Pull request merged**